qutip/tests/solver/heom/test_bofin_baths.py

```diff
@@ -463,3 +472,247 @@ def test_create(self):
                 sigma_bar_k_offset=-1,
                 tag="bath1",
             )
+
+
+class TestFitUtils:
+    cks = [1] * 5
```

Comment:
This looks unused. Could you remove it? I saw @pmenczel found quite a few unused things too. Could you be a bit more careful about leaving cruft like this lying around? It's tricky for reviewers to spot.

Reply:
Hi, All the functions inside are still used, they are just not part of a class anymore and are now global functions,, I never separated them for tests, mainly because I saw all tests were written for classes and wasn't sure if that was part of the style. I will split the tests into individual tests now. Thanks for pointing it out :)